src/components/Grid/useGridSorting.js

```diff
@@ -26,6 +26,35 @@ export const useGridSorting = (rows, columns) => {
     return <SortIcon columnKey={key} />
   }
 
+  const setSortState = (key) => {
```

Comment:
@roblevintennis couldn't figure out how to write specs mostly because the original logic of `compareBy` didn't change and when trying to write tests for `setSortState` I was beginning to test implementation lol. Let me know what you think!

Reply:
Yeah, I had a look and I don't think we can easily get coverage for `setSortState` since `testHook` which creates a parent component so we can abide by React Hooks rules of only using a hook within a top level component will rerender in an infinite loop (unsurprisingly). I think other libs might start to make that easier and Brad has an open PR which might be adding one of them. I think it's fine for now....

But, we can certainly get _some_ coverage :) I'm cooking up a PR https://github.com/getethos/ethos-design-system/pull/238 and I'll try pulling that into your list refactor 4 branch and ensure I haven't borked anything.